lineage/states/StateDistributionGamma.py

```diff
@@ -68,6 +68,8 @@ def pdf(self, x: np.ndarray):
         # Update for observed Bernoulli
         ll[np.isfinite(x[:, 0])] += sp.bernoulli.logpmf(x[np.isfinite(x[:, 0]), 0], self.params[0])
 
+        ll[x[:, 0] == -1] = np.log(1/num_states)
```

Comment:
Why do you need this? It seems like you should be able to just set the log-likelihood to 0.

Reply:
It was my first intuition, but putting the likelihood of those cells to zero will create some problems in the later steps that something will be divided by this value and threw an error.

Reply:
I have not tried setting it to a small number like epsilon though. I will try that and see if it still errors.

Reply:
This is more reason to set it to 0! Under no circumstances should the code be dividing by the log-likelihood. If a 0 causes an error here, that indicates a problem that needs to be fixed elsewhere.

Reply:
I think your recent changes to the estimators fixed that. I don't get that error anymore when I set the log-likelihood to zero, but get more cost function = inf.
I am trying to find a way to make the `minimize` run again if the cost function becomes inf.

Reply:
Hmm... where does inf first arise? Is it in the estimator? It's likely a quick fix if we can track it down.

Reply:
It is inside the `gamma_LL()` line 51 stateCommon.py (where you have `if np.isinf(outt): print(x)`). The assersion about `outt` being finite is violated.